docs/repositories-configure/configuring-code-patterns.md

```diff
@@ -103,19 +94,25 @@ Codacy will use the updated configurations on the next analysis.
     docs/repositories-configure/codacy-configuration-file.md (list of tool short names to use on the Codacy configuration file)
 -->
 
-Codacy supports configuration files for several tools. To use a configuration file for your static analysis tool:
+!!! note
+    -   Once set, Codacy uses a configuration file even if you exclude it from Codacy analysis [using the Codacy UI](ignoring-files.md) or a [Codacy configuration file](codacy-configuration-file.md#syntax-for-ignoring-files).
+
+    -   When you update pattern settings using a configuration file, Codacy may display outdated messages for issues identified previously by those patterns.<!-- TODO Can we remove this? If not, can we clarify when this happens and if this is temporary?  -->
 
-1.  Push the configuration file to the root of the branch [configured as the main branch on Codacy](managing-branches.md).
+Codacy supports configuration files for several static analysis tools to help you streamline your setup.
 
-1.  Open your repository **Code patterns** page, select the tool that will use the configuration file, and select the option **Configuration file**.
+To use a configuration file for a static analysis tool:
 
-    ![Using a configuration file](images/code-patterns-config-file.png)
+1.  Push the configuration file to the root of the [default Codacy branch](managing-branches.md).
+
+1.  Open the repository **Code patterns** page, select the tool of interest, and select the option **Configuration file**.
 
-After activating the option to use the configuration file:
+    !!! note
```

Comment:
I agree with moving this note here, but I find the original sentences more clear. 🤔 

```
-   Codacy will use the version of the configuration file **in the branch being analyzed**. For example, if you open a pull request that includes changes to the configuration file, the analysis results will take those changes into account.
-   If Codacy analyzes a branch that doesn't include the configuration file, Codacy reverts to using the code patterns configured for the tool before you selected the option **Configuration file** on the Code patterns page.
```

Reply:
OK. I'll just reword slightly to use the present tense.